### ISSUE TITLE
Frozen toolbar in Google docs

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#399]
while testing #349, I did experience again the frozen toolbar in google docs. this time I logged in as apple user, then I checked my wordlist, then I change page language from chinese to latin in lookup and the toolbar is frozen, I cannot select Options or wordlist or anything else. After deactivating alpheios and closing google docs, it was back ok.

![Screen Shot 2020-05-29 at 2 58 04 PM](https://user-images.githubusercontent.com/41396793/83262351-1ddaf080-a1bd-11ea-8908-1fdcde0a6e18.png)

**Comment by kirlat** [comment]
@monzug, on what OS did it happens? Was it macOS, Windows, or Linux?

**Comment by monzug** [comment]
chrome on Mac

**Comment by kirlat** [comment]
Thanks, I will investigate

**Comment by monzug** [comment]
I did try in latest build and I was not able to repro it. will take it to add to regression. Thanks

**Comment by monzug** [comment]
uhm, I have seen again  a frozen toolbar when logged in google docs.

**Comment by balmas** [comment]
I think I was just able to reproduce it once in your test document in Chrome on Windows, but cannot do it again.

**Comment by monzug** [comment]
again, switching from google docs to latin library in FF. activate alpheios, the toolbar is frozen
Note that wordlist is also highlighted (and it shouln't). maybe bleeding from previous session? by reloading the page it works fine
![frozen-toolbar-in-latin-FF](https://user-images.githubusercontent.com/41396793/83895323-87f51780-a720-11ea-89c2-52c54f399ad6.JPG)

**Comment by balmas** [comment]
hmm. @monzug are you usually logged into a use account when you experience this?

**Comment by monzug** [comment]
no, I am not logged in

**Comment by balmas** [comment]
Have you seen it anytime other than switching into/out of Google Docs?

**Comment by monzug** [comment]
I am not sure. I have started paying attention after experiencing few
times.

**Comment by monzug** [comment]
again a frozen toolbar in FF on MAC. installed the new build then activate alpheios on loeb classics and got the frozen toolbar after expanding it. by reloading and activate/deactivate alpheios, I realized that the page language is chinese and mouse move is checked. so maybe this needs to be investigated.

**Comment by balmas** [comment]
I tried reproducing on FF on Mac and cannot :-(

@monzug I  am a little lost on the different combinations of where this is reproducible for you. If it is only on Mac, I am tested to ask you to upgrade your OS. High Sierra is at least 2, maybe now 3, releases below the latest Mac OS.  But if it's happening on Windows too, then that is not likely to be relevant.

**Comment by monzug** [comment]
nope, only on Mac. what should I upgrade to? I do always upgrade when
prompt to.

**Comment by balmas** [comment]
Catalina is the latest version. Mojave is the one in between HIgh Sierra and Catalina. Kirill and Irina and I are all on Catalina.

**Comment by monzug** [comment]
I am on Catalina too.

**Comment by balmas** [comment]
oh :-( I thought you were still on High Sierra.

**Comment by monzug** [comment]
I mean I just installed it tonite

**Comment by balmas** [comment]
oh. ok!

**Comment by monzug** [comment]
to retest in regression
